packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -29,18 +38,49 @@ export class QuickFileOpenService implements QuickOpenModel {
     }
 
     private wsRoot: FileStat | undefined;
+    private showIgnored: boolean = false;
+    private opened: boolean = false;
 
     isEnabled(): boolean {
         return this.wsRoot !== undefined;
     }
+    isOpened(): boolean {
```

Comment:
I suggest you remove `isOpened()` and `isShowIgnored()`.  Those fields are both used only internally.

Reply:
Remove both methods